pages/getting_started/onboarding_faqs.md

```diff
@@ -191,11 +191,12 @@
 ## Pricing 
 
 1. How is the oracle price is computed?
-    - The oracle price has four parts:
-        - pricefeed daemon: side car that pulls price data from external sources and caches them for the node to use [link](https://github.com/dydxprotocol/v4-chain/tree/af8b6a46fdecc77ef154fd7b32377b4fea92b3f8/protocol/daemons/pricefeed)
-        - consensus: proposes new prices [link](https://github.com/dydxprotocol/v4-chain/tree/af8b6a46fdecc77ef154fd7b32377b4fea92b3f8/protocol/app/prepare) and validates the new prices are acceptable [link](https://github.com/dydxprotocol/v4-chain/blob/af8b6a46fdecc77ef154fd7b32377b4fea92b3f8/protocol/app/process/market_prices.go)
+    - The oracle price has five parts:
+        - Skip Protocol Sidecar: side car that pulls price data from external sources and caches them for the validator to use [link](https://docs.skip.money/slinky/integrations/dydx)
+        - Vote Extensions: Every block during the Precommit stage, all validators will submit vote extensions for what they believe the oracle price of all tracked assets should be.
+        - consensus: The block after VE are submitted, Slinky deterministically aggregates all VE from the previous block and proposes a new updated price which is voted into consensus.
```

Comment:
nit: Case for the last 3 bullets is different from the first 2. Make this consistent